### PR TITLE
explanation on StuDash

### LINKED ISSUE #6: Add Readme

### PR DESCRIPTION
**Pull request by nofarda**
I wrote an explanation about the use and benefits of our learning App.
Fixed #6

### CODE REVIEW

**Review by ravidbro — CHANGES_REQUESTED** [review]
The content is good.
You need to link the PR to an issue.
This is ".md " file that supports formatting, look at how the file looks like in the rich view with formatting and align it.

**Comment by ifireball** [comment]
Please add a more detailed PR description

**Comment by noaramon** [comment]
Fixes #6

**Comment by ravidbro** [comment]
> Fixes #6

For update like this, it's better to update the description of the PR and not as a comment

**Comment by ravidbro** [comment]
Please format it with .md syntax and look that the look makes sense.
You can see about the syntax here: https://guides.github.com/features/mastering-markdown/
You can also use: https://dillinger.io/

Nothing fancy is expected, but at least headers, brake lines, etc.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 The system makes it possible to illustrate, research and practice the material learned in a meaningful and in-depth way that is close to the students' language and increases curiosity and motivation for learning.
 StuDash includes many assessment assignments, study units, videos, simulations, presentations and more, all in a variety of subjects like: mathematics, science, Hebrew, English, history etc.
-The content is adapted to the curriculum of the Ministry of Education
-In addition, StuDash is an advanced and friendly learning environment that provides a complete and comprehensive systemic response also to the teachers.
+The content is adapted to the curriculum of the Ministry of Education!
+
+In addition, StuDash is an advanced and friendly learning environment that provides a complete and comprehensive systemic response also to the teachers."
+</br>
+## DEMO - Image
+[ofek App](https://www.easyform.co.il/SystemFiles/242/Ofek.jpg)
```
Comment (ravidbro) [inline README.md]: Embedding images can be problematic legally. 
Please put a link to a web page on the site or remove it from here.

**Review by ravidbro — CHANGES_REQUESTED** [review]

**Comment by ravidbro** [comment]
Why did you add 'StuDash' files in this PR?
It's not needed for updating the ReadMe, and also it creates a conflict

**Comment by ravidbro** [comment]
Ensure that the PR has only one commit for such a change of one file at the end of the fix.

File: README.md (hunk 1 above)
Comment (ravidbro) [reply README.md]: I meant to remove the link to the png file, not removing the README

**Comment by Gal-f** [comment]
I'm closing this PR and sent a new one with the same content at #14.
This is due to some errors we weren't able the resolve thus far.

**Pull request closed without merging**